src/views/self/view_weapons.cpp

```diff
@@ -216,12 +220,12 @@ namespace big
 			ImGui::PushItemWidth(250);
 			if (ImGui::BeginCombo("Attachments", selected_weapon_attachment.c_str()))
 			{
-				auto weapon = g_gta_data_service->weapon_by_hash(selected_weapon_hash);
+				weapon_item weapon = g_gta_data_service->weapon_by_hash(selected_weapon_hash);
 				if (!weapon.m_attachments.empty())
 				{
 					for (std::string attachment : weapon.m_attachments)
 					{
-						auto attachment_component   = g_gta_data_service->weapon_component_by_name(attachment);
+						weapon_component attachment_component   = g_gta_data_service->weapon_component_by_name(attachment);
```

Comment:
Any reason for changing the type from `auto` to `weapon_item`?

Reply:
nope, just don't like auto